src/foam/java/refinements.js

```diff
@@ -248,7 +248,10 @@ foam.CLASS({
         foam.core.PropertyInfo prop = this;
         map.put(getName(), new foam.lib.csv.FromCSVSetter() {
           public void set(foam.core.FObject obj, String str) {
+          try{
```

Comment:
Explain why this is needed and clean up the formatting.

Reply:
Done.